packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -125,6 +126,23 @@ export class InternalDropdownItem extends React.Component<InternalDropdownItemPr
     });
   }
 
+  extendChildClass(
+    children: React.ReactNode,
+    itemClass: string,
+    ref: React.ReactNode,
+    id: string,
+    additionalProps: any
+  ) {
+    return React.Children.map(children as React.ReactElement<any>, child =>
+      React.cloneElement(child, {
+        className: classNames(child.props.className, itemClass),
```

Comment:
Change to `css(child.props.className, itemClass)`

Reply:
maybe it would make sense if this method is a helper method that could be utilized by other components?

Reply:
All set! I think I subconsciously thought I was working on OpenShift.